web/ViewModels/PostcodeApiResponseViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Newtonsoft.Json;
+
+namespace BricksAndHearts.ViewModels;
+
+public class PostcodeApiResponseViewModel
```

Comment:
This isn't a view model (we're not using it in a view). I think move it into the services folder, or even inside ApiService.

Reply:
ok done